components/Collections/src/MDCCollectionViewController.m

```diff
@@ -213,8 +213,20 @@ - (CGSize)sizeWithAttribute:(UICollectionViewLayoutAttributes *)attr
 
 - (CGFloat)cellWidthAtSectionIndex:(NSInteger)section
                     collectionView:(UICollectionView *)collectionView {
-  CGFloat bounds =
-      CGRectGetWidth(UIEdgeInsetsInsetRect(collectionView.bounds, collectionView.contentInset));
+  UIEdgeInsets contentInset = collectionView.contentInset;
+  // On the iPhone X, we need to offset
+  if ([self.view respondsToSelector:@selector(safeAreaInsets)]) {
+    NSMethodSignature *adjustedContentInsetSignature =
+        [[UIScrollView class] instanceMethodSignatureForSelector:@selector(safeAreaInsets)];
```

Comment:
Shouldn't these two selectors be `@selector(adjustedContentInset)` and checking that `self.collectionView` responds? I believe all phones respond to this selector once they're running iOS 11.

Reply:
That was a mistake.
Done.